packages/version-bump/src/tests/utils.ts

```diff
@@ -48,187 +49,198 @@ export const checkForDuplicateCases = (testCases: Array<Record<string, unknown>>
+		outputVer: '5.0.0.beta',
 	},
 	{
-		type: 'decaf',
-		inputVer: '4.10.9.beta',
-		outputVer: '4.10.9.decaf',
+		type: 'major',
+		inputVer: '0.0.0',
+		outputVer: '1.0.0.rc',
 	},
 	{
-		type: 'decaf',
+		type: 'major',
 		inputVer: '0.0.0',
-		outputVer: '0.0.0.decaf',
+		outputVer: '5.0.0.rc',
+		// explicitly set the version
+		value: '5',
 	},
 	{
-		type: 'rc',
+		type: 'minor',
 		inputVer: '4.10.9.rc.011',
-		// only build shoud be incremented
-		outputVer: '4.10.9.rc.012',
+		// minor is incremented and patch is set to 0
+		outputVer: '4.11.0.rc',
 	},
 	{
-		type: 'rc',
+		type: 'minor',
 		inputVer: '4.10.9.rc',
-		// patch shoud be incremented and build set to first
-		outputVer: '4.10.10.rc.001',
+		outputVer: '4.11.0.rc',
 	},
 	{
-		type: 'rc',
-		inputVer: '4.10.9.beta',
-		// patch shoud be incremented and build set to first
-		outputVer: '4.10.10.rc.001',
+		type: 'minor',
+		inputVer: '4.3.9.beta',
+		outputVer: '4.4.0.beta',
 	},
 	{
-		type: 'rc',
+		type: 'minor',
+		inputVer: '4.8.0.beta',
+		// no change to patch when it's already 0
+		outputVer: '4.9.0.beta',
+	},
+	{
+		type: 'minor',
 		inputVer: '0.0.0',
-		// patch shoud be incremented and build set to first
-		outputVer: '0.0.1.rc.001',
+		outputVer: '0.1.0.rc',
 	},
 	{
-		type: 'rc',
-		inputVer: '4.10.9.beta.011',
-		// only build shoud be incremented
-		outputVer: '4.10.9.rc.012',
+		type: 'minor',
+		inputVer: '5.3.1',
+		outputVer: '5.5.0.rc',
+		// explicitly set the version
+		value: '5',
 	},
 	{
-		type: 'alpha',
+		type: 'patch',
 		inputVer: '4.10.9.rc.011',
-		// only build shoud be incremented
-		outputVer: '4.10.9.alpha.012',
+		// only patch shoud be incremented and build stripped
+		outputVer: '4.10.10.rc',
 	},
 	{
-		type: 'alpha',
+		type: 'patch',
 		inputVer: '4.10.9.rc',
-		// patch shoud be incremented and build set to first
-		outputVer: '4.10.10.alpha.001',
+		// only patch shoud be incremented
+		outputVer: '4.10.10.rc',
 	},
 	{
-		type: 'alpha',
-		inputVer: '4.10.9.beta',
-		// patch shoud be incremented and build set to first
-		outputVer: '4.10.10.alpha.001',
+		type: 'patch',
+		inputVer: '2.10.9.beta',
+		// patch shoud be incremented
+		outputVer: '2.10.10.beta',
 	},
 	{
-		type: 'alpha',
+		type: 'patch',
 		inputVer: '0.0.0',
-		// patch shoud be incremented and build set to first
-		outputVer: '0.0.1.alpha.001',
+		// patch shoud be incremented
+		outputVer: '0.0.1.rc',
 	},
 	{
-		type: 'beta',
-		inputVer: '4.10.9.rc.011',
-		// only build shoud be incremented
-		outputVer: '4.10.9.beta.012',
+		type: 'patch',
+		inputVer: '5.10.9.beta.011',
+		// patch shoud be incremented and build should be stripped
+		outputVer: '5.10.10.beta',
 	},
 	{
-		type: 'beta',
-		inputVer: '4.10.9.rc',
-		// patch shoud be incremented and build set to first
-		outputVer: '4.10.10.beta.001',
+		type: 'patch',
+		inputVer: '1.0.6',
+		outputVer: '1.0.5.rc',
+		// explicitly set the version
+		value: '5',
```

Comment:
this was a great addition to the logic... that wasn't possible before was it?

Reply:
No, that wasn't possible

Reply:
ya awesome... good thinking